src/components/user-list-item/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { formatDistanceStrict } from 'date-fns';
+
+export const getDateDistance = (date: Date | string | number) => {
+  const time = formatDistanceStrict(new Date(date), new Date());
+  if (time.includes('seconds') && parseInt(time) <= 30) {
+    return 'Now';
+  }
+  return time;
+};
```

Comment:
Why not use `formatDistanceToNow` instead of this function?
"Now" is not so important